packages/dd-trace/src/appsec/iast/index.js

```diff
@@ -55,6 +54,7 @@ function onIncomingHttpRequestStart (data) {
           createTransaction(rootSpan.context().toSpanId(), iastContext)
           overheadController.initializeRequestContext(iastContext)
           taintTrackingPlugin.taintHeaders(data.req.headers, iastContext)
+          iastTelemetry.onRequestStart(iastContext)
```

Comment:
`iastTelemetry.onRequestStart` should be before `taintHeaders` to have the `EXECUTED_SOURCE` metric incremented in the context's `namespace` . Otherwise, it will be incremented in `globalNamespace` and I'm not sure this is what we want to achieve.

Reply:
good catch, changed and added a test